lib/Dialect/FIRRTL/FIRRTLUtils.cpp

```diff
@@ -803,7 +803,10 @@ circt::firrtl::maybeStringToLocation(StringRef spelling, bool skipParsing,
 /// Given a type, return the corresponding lowered type for the HW dialect.
 /// Non-FIRRTL types are simply passed through. This returns a null type if it
 /// cannot be lowered.
-Type circt::firrtl::lowerType(Type type) {
+Type circt::firrtl::lowerType(
```

Comment:
This change ties this function to emission (it's doing op creation), which means it probably doesn't belong in utils anymore.